src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${partner}` : 'I am not married'}
```

Comment:
2. [CODE STYLE] - Format ternary operator operands correctly - move each operand to the separate line. This will improve the readability of your code. Here is the improved version: 
```jsx
{isMarried 
  ? `${partnerName} is my ${partner}` 
  : 'I am not married'}
```